app/services/events/record.rb

```diff
@@ -307,6 +307,26 @@ def self.record_bulk_upload_completed_event!(author:, batch:)
       new(event_type:, author:, appropriate_body: batch.appropriate_body, heading:, happened_at: Time.zone.now, metadata:).record_event!
     end
 
+    # API Token Events
+
+    def self.record_lead_provider_api_token_created_event!(author:, api_token:)
+      event_type = :lead_provider_api_token_created
```

Comment:
testing this locally we need to add:
```
lead_provider_api_token_created
lead_provider_api_token_revoked
```
to event types here: https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/models/event.rb#L2

Reply:
also worth making sure we test this

Reply:
Fixed in https://github.com/DFE-Digital/register-early-career-teachers-public/pull/632/commits/70d7f8d33333657227b7bc5fbd044eacd82f83fa